### PR TITLE
feat: polymer localize behavior

### LINKED ISSUE #30: Internationalization of central dashboard
Central dashboard must be available in both Official Languages.

### PR DESCRIPTION
**Pull request by wg102**
Resolves https://github.com/StatCan/kubeflow/issues/30

### CODE REVIEW

File: components/centraldashboard/public/components/localization-mixin.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// eslint-disable-next-line max-len
+import {AppLocalizeBehavior} from '@polymer/app-localize-behavior/app-localize-behavior.js';
+import {languages} from '../assets/i18n/languages.json';
+import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
+import {PolymerElement} from '@polymer/polymer';
+
+// eslint-disable-next-line max-len
+export default class extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
```
Comment (brendangadd) [inline components/centraldashboard/public/components/localization-mixin.js:8]: I think defining the mixin this way breaks the inheritance chaining. What happens if you take a superclass argument like the utilities mixin does and pop that in stead of `PolymerElement`?

File: components/centraldashboard/public/assets/i18n/languages.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            "dashboardView.quicklinkViewKatibText": "View Katib Studies",
+            "dashboardView.quicklinkViewKatibDesc": "Katib",
+            "dashboardView.quicklinkMetadataArtifactsText": "View Metadata Artifacts",
+            "dashboardView.quicklinkMetadataArtifactsDesc": "Artifact Store",
+            "dashboardView.headingRecentPipelines": "Recent Pipelines",
+            "dashboardView.headingRecentPipelinesRuns": "Recent Pipeline Runs",
+            "dashboardView.headingDocumentation": "Documentation",
+            "dashboardView.docItemAAWDText": "FR Advanced Analytics Workspace Docs",
+            "dashboardView.docItemAAWDDesc": "FR Helpful guides about our data and analysis tools",
+            "dashboardView.docItemAAWDLink": "https://statcan.github.io/daaas/",
```
Comment (brendangadd) [inline components/centraldashboard/public/assets/i18n/languages.json:20]: * Avoiding all-caps in camel-case abbreviations makes them clearer.
* You have a structural pattern here with the three values per doc item.

So maybe...
```
"dashboardView.docItemAawDocs.text": "...",
"dashboardView.docItemAawDocs.desc": "...",
"dashboardView.docItemAawDocs.link": "...",
```

... and so on.

File: components/centraldashboard/public/components/main-page.js [hunk 3]
```diff
@@ -45,13 +48,82 @@ import {IFRAME_LINK_PREFIX} from './iframe-link.js';
...
+export class MainPage extends utilitiesMixin(mixinBehaviors([AppLocalizeBehavior], PolymerElement)) {
     static get template() {
         const vars = {logo};
         return html([
             `<style>${css.toString()}</style>${template(vars)}`]);
     }
 
+    constructor() {
+        super();
+        this.resources = {
```
Comment (brendangadd) [inline components/centraldashboard/public/components/main-page.js:61]: I'm guessing you're leaving these as they are until your mixin approach is reviewed. :+1: So just a reminder...

**Review by brendangadd — CHANGES_REQUESTED** [review]

File: components/centraldashboard/public/components/localization-mixin.js (hunk 1 above)
Comment (wg102) [reply components/centraldashboard/public/components/localization-mixin.js]: Modified to use superclass argument instead

**Review by brendangadd — APPROVED** [review]
Beautiful

**Pull request merged**